website/templates/index.mako

```diff
@@ -1,324 +1,11 @@
-            <p>The OSF helps our students understand and apply sound data management principles to their work. And since we have easy access to all of the files the students are working with, it greatly enhances our ability to offer them constructive guidance.<br/><small><em>Richard Ball, Professor of Economics, Haverford College</em></small></em></small></p>
-          </div>
-        </div>
-
-        <div class="row hidden-xs hidden-sm">
-          <div class="col-md-7 col-md-offset-2">
-            <h3>Version control makes life easier</h3>
-            <p>The OSF makes version control effortless. My PI, my lab mates, and I have access to previous versions of a file at any time&#151;and the most current version is always readily available.<br/><small><em>Erica Baranski, PhD Student, Social and Personality Psychology Funder Lab, UC Riverside</em></small></em></small></p>
-          </div>
-          <div class="col-md-3">
-            <img src="/static/img/front-page/user3.jpg" class="img-circle img-responsive" alt="Erica Baranski" />
-          </div>
-        </div>
-
-        <div class="row hidden-xs hidden-sm">
-          <div class="col-md-3">
-            <img src="/static/img/front-page/user4.jpg" class="img-circle img-responsive" alt="Anne Allison" />
-          </div>
-          <div class="col-md-7">
-            <h3>A centralized hub of information</h3>
-            <p>The OSF creates a centralized hub of information where I can oversee a diversity of research projects across multiple classes. The centralization, organization and anywhere-access save me the time and energy necessary for managing these projects.<br/><small><em>Anne Allison, Associate Professor of Biology at Piedmont Virginia Community College</em></small></em></small></p>
-          </div>
-        </div>
-
-
-      </div>
-    </div>
-    <div class="space-top space-bottom feature-6">
-      <div class="container">
-        <div class="row">
-          <div class="col-md-8">
-            <h2><strong>Free and open source.</strong></h2>
-            <h4>The OSF is a public good built to support your research.</h4>
-            <a href="#" class="btn btn-info btn-lg">Get started</a>
-          </div>
-          <div class="col-md-4 hidden-xs hidden-sm">
-            <div id="logo">
-              <div class="circle" id="circle-1"><span></span></div>
-              <div class="circle" id="circle-2"><span></span></div>
-              <div class="circle" id="circle-3"><span></span></div>
-              <div class="circle" id="circle-4"><span></span></div>
-              <div class="circle" id="circle-5"><span></span></div>
-              <div class="circle" id="circle-6"><span></span></div>
-              <div class="circle" id="circle-7"><span></span></div>
-              <div class="circle" id="circle-8"><span></span></div>
-            </div>
-          </div>
-        </div>
-      </div>
-
-    </div>
-
-
+<%def name="logged_in(name)">
+    %if name != '':
```

Comment:
Need to learn some Mako ... what `name` is this? The function-ish parameter just above, or the HTML-ish attribute value just above (the result of the function-ish call?)?